app/views/page_version/show.html.slim

```diff
@@ -22,6 +22,7 @@ table.diff-versions(data-fullheight='{ "bottom": 30, "cssrule": "min-height" }')
           =link_to(page_version_show_path(:page_version_id => version.id), class: ('selected' if version == @selected_version))
             =time_tag(version.created_on)
               =l(version.created_on.to_date)
+              = version.status && render_status_icon(version.status)
```

Comment:
Let's pull the status from the messages using something like `t("page.edit.page_#{version.status}")` here so that we can see a more attractive (and translated) status on hover.